package.json

```diff
@@ -46,6 +47,7 @@
     "eslint-plugin-import": "^2.20.1",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.18.3",
-    "eslint-plugin-react-hooks": "^1.7.0"
+    "eslint-plugin-react-hooks": "^1.7.0",
+    "serve": "^11.3.0"
```

Comment:
Are we installing this incidentally? I guess it's handy for development..

Reply:
Let's not though.